src/stub.js

```diff
@@ -0,0 +1,10 @@
+import m3u8 from './m3u8';
```

Comment:
I do wish this file name was more descriptive. Are you against videojs-hls.js for the project entry point?